tests/project/__snapshots__/tsconf.tests.ts.snap

```diff
@@ -0,0 +1,503 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I think this should be removed.

Reply:
This line (`Jest Snapshot v1`) do you mean? Or the snapshot?

It's auto-generated by Jest with this line:

https://github.com/darkreader/darkreader/pull/8266/commits/eb9c0991bae74325d940ee9b32b3b045bc307db5#diff-c74ba150027d107ea4b48c58571626d22a410ef0c3d4a7459c91e416b9fd48e9R38

Reply:
Can it be gitignored? It isn't really a source code file if it's autogenerated.

Reply:
No, because the point of the snapshot is to mark the expected output:

https://jestjs.io/docs/snapshot-testing

It's equivalent to just doing this:

```
expect(got).toEqualObject({
  compilerOptions: {
    ...
  },
  ...
  ...
})
```

.. but it's easier to maintain because you can just press `u` in a watch session, or pass `--updateSnapshot` if a change comes up and you accept the change. 

To be honest, I wrote these as temporary "pins" so I didn't affect anything while making these changes. Unless you mentioned that you wanted these `tests/project` tests to stay, I was going to delete them after we felt confident that the upgrades were working.

Reply:
> if it's autogenerated.

It's only autogenerated the first time, then it requires approval to autogenerate again.